test/1_stdlib/ArrayTrapsObjC.swift.gyb

```diff
@@ -0,0 +1,197 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+// RUN: %S/../../utils/gyb %s -o %t/ArrayTraps.swift
+// RUN: %S/../../utils/line-directive %t/ArrayTraps.swift -- %target-build-swift %t/ArrayTraps.swift -o %t/a.out_Debug
+// RUN: %S/../../utils/line-directive %t/ArrayTraps.swift -- %target-build-swift %t/ArrayTraps.swift -o %t/a.out_Release -O
+//
+// RUN: %S/../../utils/line-directive %t/ArrayTraps.swift -- %target-run %t/a.out_Debug
+// RUN: %S/../../utils/line-directive %t/ArrayTraps.swift -- %target-run %t/a.out_Release
+// REQUIRES: executable_test
+// REQUIRES: objc_interop
+
+import StdlibUnittest
+
+// Also import modules which are used by StdlibUnittest internally. This
+// workaround is needed to link all required libraries in case we compile
+// StdlibUnittest with -sil-serialize-all.
+import SwiftPrivate
+#if _runtime(_ObjC)
+import ObjectiveC
```

Comment:
Interesting note:

I went through the source tree and killed as many imports of ObjectiveC that fit this pattern as I could find, about 20 or so. I then ran `ninja check-swift-all-optimize` in my existing OS X swift build. It rebuilt about 500 files, ran validation tests, and all of them passed without issue.

The original build was produced with `build-script -Rt`